coderadar-graph/src/main/java/io/reflectoring/coderadar/graph/query/repository/ContributorQueryRepository.java

```diff
@@ -0,0 +1,23 @@
+package io.reflectoring.coderadar.graph.query.repository;
+
+import io.reflectoring.coderadar.graph.contributor.domain.ContributorEntity;
+import io.reflectoring.coderadar.graph.query.domain.ContributorsForFileQueryResult;
+import java.util.List;
+import org.springframework.data.neo4j.annotation.Query;
+import org.springframework.data.neo4j.repository.Neo4jRepository;
+import org.springframework.lang.NonNull;
+
+public interface ContributorQueryRepository extends Neo4jRepository<ContributorEntity, Long> {
+  @Query(
```

Comment:
This query is (likely) only correct for the latest commit because of the way renames are handled.